examples/config files - basic/3 connector-ldap.yml

```diff
@@ -77,6 +81,9 @@ group_filter_format: "(&(|(objectCategory=group)(objectClass=groupOfNames)(objec
 # that is contained in the group.  If you want indirect containment, then
 # use this value instead of the default:
 # group_member_filter_format: "(memberOf:1.2.840.113556.1.4.1941:={group_dn})"
+# If enabled, will be used whenever "--users group" or "--users mapped" arguments 
+# are used and will filter more the result of all_users_filter - if set,
+# otherwise all users from directory's base_dn would make it's base.
```

Comment:
Again, this is misleading and should be removed.  We use this whenever we need the content of a group, whether specified in the command line or in the group mapping.  I believe the only confusing documentation in this file is the description of the all_users_filter; see above.